src/fluree/db/query/analytical_parse.cljc

```diff
@@ -208,11 +208,11 @@
   (some (fn [{:keys [s o type] :as where-smt}]
           (or (= (:variable o) variable)
               (= (:variable s) variable)
-              (cond
-                (= type :optional) (variable-in-where? variable (:where where-smt))
-                (= type :binding) (= (:variable where-smt) variable)
-                (= type :union) (or (variable-in-where? variable (first (:where where-smt)))
-                          (variable-in-where? variable (second (:where where-smt)))))))
+              (case type
```

Comment:
Is this `case` exhaustive? I sometimes prefer `cond` unless I know that the `case` is exhaustive, since it will blow up on a missing case.

Reply:
Ah, that's probably what @bplatz was referring to. I think we want it to ignore other types, so I'll add a default that does that.